src/main/java/Eliz/Eliz.java

```diff
@@ -0,0 +1,331 @@
+        FileWriter fw = new FileWriter(filePath,true);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+    /*Deals with the input and calling methods to configure inputs*/
+    public static void getInput(String line, ArrayList<Task> tasks, String filePath) throws ElizException, FileSystemNotFoundException, IOException {
+        String[] breakTaskNames = line.split(" ", 2);
+
+//        if (line.substring(line.length() - 1) == " ") { //in the case that the last character is a space
+//            breakTaskNames[0] = line;
+//            breakTaskNames[1] = null;
+//        } else {
+//            breakTaskNames = line.split(" ", 2);
+//        }
+
+        if ((breakTaskNames.length < 2)){
+            String firstWord = breakTaskNames[0];
+            switch (firstWord) {
+            case "list" : printTasks(tasks);
+                break;
+            case "todo":
+            case "deadline":
+            case "event":
+                System.out.println("OOPS!!! The description of a " + firstWord + " cannot be empty.");
+                throw new ElizException();
+            case "/help":
+                displayCommands();
+                break;
+            default: //for mark, default, other weird spelling words
+                System.out.println("OOPS!!! I'm sorry but I do not understand what you mean");
+                throw new ElizException();
+            }
+        } else { //length of breakTaskNames is 2
+            //correct inputs for todo, event, deadline and delete
+            String firstWord = breakTaskNames[0];
+            switch (firstWord) {
+            case "mark":
+            case "unmark":
+                Eliz.markOrUnmark(line, tasks);
+                break;
+            case "delete":
+                Eliz.delete(line, tasks);
+                break;
+            case "todo":
+            case "event":
+            case "deadline":
+//            System.out.println("Index 0: " + breakTaskNames[0]);
+//            System.out.println("Index 1: " + breakTaskNames[1]);
+//            boolean isEmpty = false;
+//            if (breakTaskNames[1] == null) {
+//                isEmpty = true;
+//            }
+//            System.out.println(isEmpty);
+                if (breakTaskNames[1] == null) {
+                    System.out.println("OOPS!!! The description of a " + breakTaskNames[0] + " cannot be empty.");
+                    throw new ElizException();
+                } else {
+                    Task t = createTask(line);
+                    tasks.add(t);
+                    System.out.println("Got it. I've added this task ");
+                    System.out.println(t);
+                    System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+                    /*every time an object is added or edited, append to file*/
+                    try {
+                        writeToFile(filePath, tasks.get(tasks.size()-1).getTaskType() + "|"
```

Comment:
This line of code has too many parameters that decrease the code readability. How about passing in a Task object as a parameter and letting the method unpack its values?